.github/workflows/build_macos.yml

```diff
@@ -0,0 +1,52 @@
+name: Build (macOS)
+
+on: [push, pull_request]
+
+jobs:
+  macOS:
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        configuration: ["Debug", "Release"]
+    steps:
+    - name: Define Build Options
+      shell: python
+      run: |
+        if "${{ matrix.configuration }}" == "Debug":
+          print('Configuring for Debug')
+          print('::set-env name=BUILD_PARAM::')
+          print('::set-env name=ARTIFACT_NAME::xqemu-debug')
+        else:
+          print('Configuring for Release')
+          print('::set-env name=BUILD_PARAM::--release')
+          print('::set-env name=ARTIFACT_NAME::xqemu-release')
+    - name: Clone Tree
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install Dependencies
+      run: |
+        brew update
+        brew install \
+          glib \
+          pixman \
+          sdl2 \
+          libepoxy \
+          coreutils \
+          pkg-config \
+          ccache
+    - name: Initialize ccache
```

Comment:
Should probably be "Initialize Compiler Cache" to make use of `ccache` internal, but more consistently keeping "Compiler Cache" in the human readable parts.

Reply:
Ok